package.json

```diff
@@ -38,10 +38,10 @@
     "package.json"
   ],
   "peerDependencies": {
-    "firebase": "2.x"
+    "firebase": "^2.4.2"
   },
   "dependencies": {
-    "firebase": "^2.4.1",
+    "firebase": "^2.4.2",
```

Comment:
If you put something both as a `peerDependency` and a `dependency` what happens? I'm assuming we want one or the other but not both here.